src/pages/Lending/hooks/useLendingSupportedAssets/index.ts

```diff
@@ -31,7 +37,9 @@ export const useLendingSupportedAssets = ({ type }: { type: 'collateral' | 'borr
     // @lukemorales/query-key-factory only returns queryFn and queryKey - all others will be ignored in the returned object
     // Infinity staleTime as we handle halted state JIT
     staleTime: Infinity,
-    select: pools => pools.filter(pool => pool.status === 'Available'),
+    select: statusFilter.length
+      ? pools => pools.filter(pool => statusFilter.includes(pool.status))
+      : undefined,
```

Comment:
suggestion: pass an optional status filter instead, meaning all if we don't pass a filter

Reply:
[8adb4bc](https://github.com/shapeshift/web/pull/7191/commits/8adb4bc5d03daab4737af445ebe41feedd9d88de)